src/nunit.xamarin/nunit.xamarin.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Company>NUnit</Company>
+    <Product>NUnit 3 Xamarin Runner</Product>
+    <Copyright>Copyright (C) 2018, Rob Prouse</Copyright>
+    <Title>NUnit.Xamarin</Title>
+    <Description>NUnit 3 Test Runner for Xamarin</Description>
```

Comment:
Title and Description are both in here twice - I think you want the bottom pair? 🙂

Reply:
We've lost a few things vs the old nuspec here. I think we probably want some of them back in, at least.

- minclientversion
- tags
- language
- licence url
- project url
- icon url
- requirelicenceacceptance

Reply:
Will fix. Odd, I know I added all those extra fields, not sure how they got lost 😕